aw_datastore/datastore.py

```diff
@@ -56,8 +57,10 @@ def __init__(self, datastore: Datastore, bucket_id: str) -> None:
     def metadata(self) -> dict:
         return self.ds.storage_strategy.get_metadata(self.bucket_id)
 
-    def get(self, limit: int = 10**4,
+    def get(self, limit: int = -1,
             starttime: datetime=None, endtime: datetime=None) -> List[Event]:
+        if limit < 0:
+            limit = 10**4
```

Comment:
Since it's in the datastore and not in a storage strategy (which handles limits aswell), i simply removed this line and it still works fine.